karma.conf.js

```diff
@@ -17,7 +17,7 @@ const browserPlatformPairs = {
 };
 
 module.exports = config => {
-  let bundleDirpath;
+  let bundleDirpath = path.join(baseBundleDirpath, hostname);
```

Comment:
previously, this would have been added to the `cfg` object before it was defined.  now, we only add it if creating the directory succeeds.

maybe we should ignore a `mkdir` failure and just use the tmp dir...  I don't want to run the build again 😜